c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -110,6 +110,8 @@ constexpr size_t kMinLargeAlloc =
 constexpr size_t kRoundLarge = 2097152; // round up large allocations to 2 MiB
 constexpr size_t kRoundUpPowerOfTwoIntervals = 16;
 
+void local_raw_delete(void* ptr);
```

Comment:
nit: don't think this is needed anymore, though it isn't doing any harm to be pre-declared...